### ISSUE TITLE
Make the Storybook show an additional render of all components with the 'Show button text labels' enabled

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#81660]
## What problem does this address?

https://core.trac.wordpress.org/ticket/65883 is only the last case where new blocks haven't been designed and tested with the 'Show button text labels' preference enabled.

In the history of this project I can't count the number of times when this preference was overlooked and;
- Either the block toolbar didn't show a text label with the preference enabled
- Or the provided labels were too long and contributed to make the block toolbar or other piaces of the UI  incredibly long, thus breaking the layout.

The fundamental problem is that few contributors and designers are aware or remember this feature exists.

Basically, every new UI should provide _two_ designs: one for the preference disabled and one for the preference enabled.

Then, each new feature, new UI, new block should be tezted also with this preference enabled.

For components, a good start would be the Storybook/

## What is your proposed solution?

I'm not an expert at all when it comes to the Storybook but I'm wondering if there is any way to show an additional render of all components with the 'Show button text labels' preference enabled.
This would greatly help improving the user experience for users who use this preference and would help avoiding layout breakages.

For example, for the Tabs block, an additional render of the component could be shown belot the regular one. There is enough space for it:

<img width="1248" height="713" alt="Image" src="https://github.com/user-attachments/assets/ca2276ee-9d3d-4dfe-9607-1a72eab119ff" />
